### PR TITLE
Set up description app

### PR DESCRIPTION
**Pull request by Yoavsc129**
The description/models.py HouseID field will be added after the House app will be merged. Tests will also be added

### CODE REVIEW

**Comment by ronyyosef** [comment]
Looks good
Please add tests

File: description/models.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django.db import models
+from django.utils import timezone
+
+
+class Description(models.Model):
+    date = models.DateTimeField(default=timezone.now)
+    text = models.TextField()
+    # house_id = models.ForeignKey('user_id', on_delete=models.CASCADE)
```
Comment (ShellyGolden) [inline description/models.py:8]: Hi
It will be good to add this function

def __str__(self):
        return self.choice_text

The reason: It’s important to add [__str__()] methods to your models, not only for your own convenience when dealing with the interactive prompt, but also because objects’ representations are used throughout Django’s automatically-generated admin. (https://docs.djangoproject.com/en/4.0/intro/tutorial02/)

**Review by LiorKGOW — APPROVED** [review]
Looks nice @Yoavsc129

**Review by shlior7 — COMMENTED** [review]
Q's:
1) what is the main purpose of a description that is aware of time of change for us?
2) Is description the right name for this feature what is it describing?
3) if it's one description for a house shouldn't it just be in the House app as a data model?

**Review by ronyyosef — APPROVED** [review]

**Review by LiorKGOW — APPROVED** [review]

**Comment by svatares** [comment]
@Yoavsc129 The name of the app isn't clear. I would suggest thinking about it again and renaming it. It is not clear what it should do and how it is connected to the bigger application you are creating.
Also, please add tests, even if it's a simple test that verifies the field is not empty or something like that.

**Comment by shlior7** [comment]
> The name of the app isn't clear

I suggest `Story` sounds more related to the goal of this feature

**Comment by svatares** [comment]
@Yoavsc129 please
1. Resolve conflicts
2. Squash commits, no need for 2 commits here
3. Update PR title according to new standard
